Update experiment-learnings.md (#346)
amplitude/amplitude-docs: content/collections/workflow/en/experiment-learnings.md

  
    To learn more about exposure events, see [this article in the Amplitude Developer Center](/docs/feature-experiment/under-the-hood/event-tracking).  
      

>     You can see the chart definition by clicking on the "Chart Controls" button.

Reviewer: ```suggestion
Click _Chart Controls_ to see the chart definition. 
```
